readme.md

```diff
@@ -42,6 +42,18 @@ ignore             do not copy files into App whose filenames regex .match this
 prune              runs `npm prune --production` on the app
 asar               packages the source code within your app into an archive
 sign               should contain the identity to be used when running `codesign` (OS X only)
+version-string     This is a hash which holds the verision details of the generated exe (windows only).
+                   These are the following keys it supports
```

Comment:
More consistent description, IMO:

> should contain a hash of the application metadata to be embedded into the executable (Windows only). Keys supported:

Opinions on whether the keys should be prefixed with `*` or `-` to visually indicate a list?